code/modules/research/xenobiology/xenobio_camera.dm

```diff
@@ -26,10 +26,16 @@
 	var/datum/action/innate/slime_pick_up/slime_up_action = new
 	var/datum/action/innate/feed_slime/feed_slime_action = new
 	var/datum/action/innate/monkey_recycle/monkey_recycle_action = new
+	var/datum/action/innate/transfer_potion/transfer_potion_action = new
+	var/datum/action/innate/hotkey_help/hotkey_help_action = new
+	var/datum/action/innate/slime_scan/slime_scan_action = new
 
 	var/list/stored_slimes = list()
 	var/max_slimes = 5
 	var/monkeys = 0
+	var/obj/item/slimepotion/slime/current_potion
+	var/obj/item/slime_scanner/scanner = new /obj/item/slime_scanner
```

Comment:
why use a scanner at all? Just having the `slime_scan` proc is...good enough.

Reply:
we don't have one. You want me to make one?

Reply:
It'd be best if it was refactored.

Reply:
refactored being yes make one right